pages/a/ms-azure-guest-post.html

```diff
@@ -13,47 +13,57 @@
 
-Our aims were to:
+We specifically wanted to:
 
-- See how Crate would distribute shards and partitions across such a large cluster
-- Examine the overhead added by a the cluster management
-- See what speeds the cluster could write data at
-- Another
-- And another
+- Know how Crate would distribute shards and partitions across such a large cluster.
+- Examine the overhead added by a the cluster management.
+- See at what speeds the cluster could write data.
 
-Microsoft Azure were generously lent us the use of 8000 cores as the infrastructure for this test and engineer time to guide us through problems and questions we might have.
+Microsoft Azure generously lent us the use of 8000 cores as the infrastructure for this test and engineer time to guide us through problems and questions we might have.
 
-Microsoft hoped to prove that their Azure platform was reliable enough for a project of such rapid scaling and resource intensive.
+Microsoft wanted to show that their Azure platform was reliable and capable enough for a project that would scale so rapidly and be so resource intesive.
 
-So, what happened and how did we do it?
+## Set Up
 
-We aimed to create a 1001 node Crate cluster and xxx. Setting up such a cluster on any kind of infrastructure requires planning and whilst cloud hosts simplify this process, understanding how to translate your requirements into their paradigms requires research and experimentation.
+We aimed to create a 1001 node Crate cluster ready for data import and manipulation. Setting up such a cluster on any kind of infrastructure requires planning and whilst cloud hosts simplify this process, understanding how to translate your requirements into their paradigms requires research and experimentation.
 
-Azure has several concepts that we needed to enable and configure to get our cluster started:
+Azure has several concepts that we needed to understand and make use of to get our cluster started:
 
 - **Resource Group**: A 'container' that holds related resources of all types for an application.
 - **Storage Account**: A data storage solution allocated to you that can contain Blob, Table, Queue, and File data. Can be stored on SSDs or HDDs.
-- **Virtual Network**: A representation of a network in the cloud, where you can define DHCP address blocks, DNS settings, security policies, and routing.
+- **Virtual Network (vnet)**: A representation of a network in the cloud, where you can define DHCP address blocks, DNS settings, security policies, and routing.
 - **Subnet**: Further segmentation for virtual networks.
 - **Virtual Machine (VM)**: Represents a particular operating system or application stack.
 - **Network Interface (NIC)**: Represents a network interface that can be associated to one or more virtual machines (VM).
```

Comment:
a single NIC can only be assigned to a single VM
but a VM can have multiple NICs assigned